tensorflow_addons/options.py

```diff
@@ -0,0 +1,49 @@
+import os
+import platform
+import warnings
+import traceback
+
+try:
+    TF_ADDONS_PY_OPS = bool(int(os.environ["TF_ADDONS_PY_OPS"]))
+except KeyError:
+    if platform.system() == "Linux":
+        TF_ADDONS_PY_OPS = False
+    else:
+        TF_ADDONS_PY_OPS = True
+
+
+FALLBACK_WARNING_TEMPLATE = """{}
+
+The {} C++/CUDA custom op could not be loaded.
+For this reason, Addons will fallback to an implementation written
+in Python with public TensorFlow ops. There worst you might experience with
+this is a moderate slowdown on GPU. There can be multiple
```

Comment:
C++ custom ops are still faster than their python op equivalents from what we've seen in profiling GELU, so this isn't limited to GPU

Reply:
Do we have benchmarks? Nice! 😃  I'm really interested, benchmarking was an issue last time we needed it in #913 . Can you link the issue/PR?

Reply:
https://github.com/tensorflow/tensorflow/pull/33945#issue-336001657

Reply:
It seems those benchmarks were only run on gpu. Furthermore, it's missing the `.numpy()` to force the execution of the ops because gpu ops are executed lazily. Benchmarking is kind of hard as we saw in #913 . I really hope we have a proper benchmarking tool in the future.

Reply:
Actually, for activation functions, for big tensors (to remove the overhead of function calls) it seems pure python (even eager mode!) is faster. XLA is much much faster.

See https://colab.research.google.com/drive/1LTx3vMpA1fLCESKl-_WrLIp0Fq1zYZL0

It might be worth opening an issue to investigate if we should drop some C++ implementations because they give no benefit at all (apart from being slightly faster for small tensors).

Reply:
Nice find! I think it's a great idea but we should look into standardizing our methods of benchmarking. I will bring it up at the next monthly meeting to see if TF core team has standard process  (though this is not blocking us from moving forward)

As @WindQAQ mentioned when testing GPU we'll want to make sure we're not benchmarking the time to move from GPU back to CPU which `.numpy()` will do:
https://colab.research.google.com/drive/12ende9xXMSywP2lOKWrFJBwaDDHbkYXh

But the tests you posted above are on CPU.

Reply:
Yeah, we've no way to actually benchmark for real on gpu. That's a shame. Either we measure the time moving back and forth between the gpu and cpu with `.numpy()`, either the ops are executed lazily and you have enormous tensors processed in nanoseconds which is not realistic at all XD

Reply:
Current numbers tend to agree with what is currently in the readme, I'll minimize this section. We can always change what is said if #1156 gives us new elements.